Library/Homebrew/formula_installer.rb

```diff
@@ -251,6 +251,7 @@ def install
         opoo "Bottle installation failed: building from source."
         raise BuildToolsError, [formula] unless DevelopmentTools.installed?
       else
+        puts @requirement_messages
```

Comment:
`@requirement_messages` is an `Array` and AFAIK `puts []` outputs nothing (no newline).
Therefore, this code already works as you suggested.